.github/workflows/codeql-java-analysis.yml

```diff
@@ -70,17 +70,34 @@ jobs:
         # queries: ./path/to/local/query, your-org/your-repo/queries@main
         queries: +security-and-quality
 
+    - name: 'Setup Temurin JDK 8, 11, 17 & 21'
+      uses: actions/setup-java@387ac29b308b003ca37ba93a6cab5eb57c8f5f93 # v4.0.0
+      with:
+        distribution: 'temurin'
+        java-version: |
+          8
```

Comment:
There was a [discussion](https://lists.apache.org/thread/bd39zhk655pgzfctq763vp3z4xrjpx58) recently at dev@ about dropping the support for Java 8.

Reply:
In this change set I went for just updating the build environment while maintaining backwards compatibility. As you can see it is possible.
Although fully dropping Java 8 would make it all a lot easier.